com.chmouel.gnomeNextMeetingApplet.yaml

```diff
@@ -0,0 +1,125 @@
+      - "-DENABLE_GOA=ON"
+      - "-DENABLE_GOOGLE=OFF"
+      - "-DENABLE_GTK=ON"
+      - "-DENABLE_GTK_DOC=OFF"
+      - "-DENABLE_INSTALLED_TESTS=OFF"
+      - "-DENABLE_INTROSPECTION=ON"
+      - "-DENABLE_VALA_BINDINGS=ON"
+      - "-DENABLE_WEATHER=OFF"
+      - "-DWITH_LIBDB=OFF"
+      - "-DWITH_OPENLDAP=OFF"
+    sources:
+      - type: archive
+        url: https://download.gnome.org/sources/evolution-data-server/3.40/evolution-data-server-3.40.4.tar.xz
+        sha256: 87c185f18c37270e3611981f19bd9221ac974c807462c8dce90bea08712c5800
+    cleanup:
+      - /lib/cmake
+      - /lib/evolution-data-server/*-backends
+      - /libexec
+      - /share/dbus-1/services
+
+  - name: python-deps
+    buildsystem: simple
+    build-commands:
+      - pip3 install --exists-action=i --no-index --find-links="file://${PWD}" --prefix=${FLATPAK_DEST} --no-build-isolation pyyaml dbus-python humanize pycairo pygobject python-dateutil six
+    sources:
+      - sha256: 98c11c755a16ef6c5673c22ca94a3802a7df4746a0853a70b6fae8b9f5cac206
+        type: file
+        url: https://files.pythonhosted.org/packages/40/d0/adc9ab25f68d9382f32c355627423a32135efaa5aaeebd6dec6d19e6b0af/poetry-core-1.0.7.tar.gz
+      - sha256: 607774cbba28732bfa802b54baa7484215f530991055bb562efbed5b2f20a45e
+        type: file
+        url: https://files.pythonhosted.org/packages/a0/a4/d63f2d7597e1a4b55aa3b4d6c5b029991d3b824b5bd331af8d4ab1ed687d/PyYAML-5.4.1.tar.gz
+      - sha256: 92bdd1e68b45596c833307a5ff4b217ee6929a1502f5341bae28fd120acf7260
+        type: file
+        url: https://files.pythonhosted.org/packages/b1/5c/ccfc167485806c1936f7d3ba97db6c448d0089c5746ba105b6eb22dba60e/dbus-python-1.2.18.tar.gz
+      - sha256: a6f7cc1597db69a4e571ad5e19b4da07ee871da5a9de2b233dbfab02d98e9754
+        type: file
+        url: https://files.pythonhosted.org/packages/46/a8/74f8bf12c7d93bb2e111d13bae198996032d9a53dbbfc3bf4837a466cbe9/humanize-3.13.1-py3-none-any.whl
+      - sha256: 1ee72b035b21a475e1ed648e26541b04e5d7e753d75ca79de8c583b25785531b
+        type: file
+        url: https://files.pythonhosted.org/packages/bc/3f/64e6e066d163fbcf13213f9eeda0fc83376243335ea46a66cefd70d62e8f/pycairo-1.20.1.tar.gz
+      - sha256: b9803991ec0b0b4175e81fee0ad46090fa7af438fe169348a9b18ae53447afcd
+        type: file
+        url: https://files.pythonhosted.org/packages/cc/72/48cfdd7a0caf3c27f392d2657731ac6f7c3c1c0a60bfeba3e1ba9ffa7ba9/PyGObject-3.42.0.tar.gz
+      - sha256: 961d03dc3453ebbc59dbdea9e4e11c5651520a876d0f4db161e8674aae935da9
+        type: file
+        url: https://files.pythonhosted.org/packages/36/7a/87837f39d0296e723bb9b62bbb257d0355c7f6128853c78955f57342a56d/python_dateutil-2.8.2-py2.py3-none-any.whl
+      - sha256: 8abb2f1d86890a2dfb989f9a77cfcfd3e47c2a354b01111771326f8aa26e0254
+        type: file
+        url: https://files.pythonhosted.org/packages/d9/5a/e7c31adbe875f2abbb91bd84cf2dc52d792b5a01506781dbcf25c91daf11/six-1.16.0-py2.py3-none-any.whl
+
+  - release.yaml
```

Comment:
why is this a different file?

Reply:
this allows me to switch them to a dev.yaml which has dir when doing dev and [it's generated](https://github.com/chmouel/gnome-next-meeting-applet/blob/main/packaging/flatpak/update-release.sh) on release with the artefacts/sha, would you rather me to concat them?